javascript/javascript3/week3/promise/promise.js

```diff
@@ -0,0 +1,39 @@
+function user1() {
+    return fetch('https://api.github.com/search/repositories?q=user:Mekdes1')
+        .then(response => response.json())
+}
+function user2() {
+    return fetch('https://api.github.com/search/repositories?q=user:mag-da-lenka')
+        .then(response => response.json())
+}
+function user3() {
+    return fetch('https://api.github.com/search/repositories?q=user:bengu-thon-mai-mochi')
+        .then(response => response.json())
+}
+const promise1 = user1()
```

Comment:
user1(), user2() and user3() are small and only used once => you shouldn't need functions for them